examples/flyio/fly.toml

```diff
@@ -0,0 +1,38 @@
+# fly.toml file generated for quiet-paper-6634 on 2023-02-24T18:49:35+01:00
+
+app = "quiet-paper-6634"
```

Comment:
Not sure if to put instructions to run fly launch to generate this file and then fly deploy 🤔